### PR TITLE
fixed: create-abell-app not working in windows

### PR DESCRIPTION
**Pull request by PiyushSuthar**
Changed
```js
execSync(`git clone ${template} ${projectName}`, execOptions)
```
to 
```js
execSync(`git clone ${template} ${projectName}`)
```
And added 
```js
process.exit(-1);
```
To automatically exit the script. 😄

### CODE REVIEW

**Review by saurabhdaware — APPROVED** [review]
Awesome! Thank you so much Piyush

File: bin/create-abell-app.js [hunk 1]
```diff
@@ -74,6 +74,8 @@ program
     console.log(`\n${colors.green('>')} Finished Installing Dependencies 🤠`);
     console.log(`${colors.green('>')} Successfully Created ${projectName} 🚀`);
     console.log(`\n${colors.green('>>')} \`cd ${projectName}\` and \`${installCommand === 'yarn' ? 'yarn' : 'npm run'} dev\` to run dev-server ✨\n\n`);
+    // Exit
+    process.exit(-1);
```
Comment (pantharshit00) [inline bin/create-abell-app.js:78]: This should be 0

Negative exit code doesn't exit. This might bite us later.

**Review by pantharshit00 — COMMENTED** [review]
@saurabhdaware maybe fix it real quick in master

File: bin/create-abell-app.js (hunk 1 above)
Comment (saurabhdaware) [reply bin/create-abell-app.js:78]: yeah I've fixed it. I was wondering why it worked for Piyush.

**Pull request merged**